.github/actions/run-cypress-tests/action.yml

```diff
@@ -63,6 +63,12 @@ runs:
         fi
       shell: bash
 
+    - name: Compile OpenSearch Dashboards
+      run: |
+        cd OpenSearch-Dashboards
+        node scripts/build_opensearch_dashboards_platform_plugins --no-examples --workers=10 --verbose
```

Comment:
With verbose output can we determine where the additional time is occurring during compilation?

Reply:
```
 debg [visTypeXy] state = "compiler success" after 9.6 sec
 debg [apmOss] state = "compiler success" after 14.9 sec
 debg [dataSource] state = "compiler success" after 15.5 sec
 debg [urlForwarding] state = "compiler success" after 71.5 sec
 debg [tileMap] state = "compiler success" after 88.9 sec
 debg [bfetch] state = "compiler success" after 105.6 sec
 debg [contentManagement] state = "compiler success" after 131 sec
 debg [uiActions] state = "compiler success" after 148.1 sec
 debg [expressions] state = "compiler success" after 207.6 sec
 debg [managementOverview] state = "compiler success" after 224.1 sec
 debg [embeddable] state = "compiler success" after 254.1 sec
 debg [mapsLegacy] state = "compiler success" after 257.1 sec
 debg [share] state = "compiler success" after 261.9 sec
 debg [devTools] state = "compiler success" after 272.2 sec
 debg [opensearchDashboardsLegacy] state = "compiler success" after 290.7 sec
 debg [advancedSettings] state = "compiler success" after 296.2 sec
 debg [regionMap] state = "compiler success" after 301.7 sec
 debg [visTypeTable] state = "compiler success" after 303.8 sec
 debg [queryEnhancements] state = "compiler success" after 313.7 sec
 debg [visTypeTagcloud] state = "compiler success" after 316.8 sec
 debg [opensearchDashboardsUtils] state = "compiler success" after 320.4 sec
 debg [console] state = "compiler success" after 325.5 sec
 debg [inspector] state = "compiler success" after 328.4 sec
 debg [visTypeMarkdown] state = "compiler success" after 338.6 sec
 debg [visTypeVislib] state = "compiler success" after 339.8 sec
 warn worker stderr [BABEL] Note: The code generator has deoptimised the styling of /home/runner/work/security-dashboards-plugin/security-dashboards-plugin/OpenSearch-Dashboards/src/plugins/data/public/antlr/opensearch_sql/.generated/OpenSearchSQLParser.ts as it exceeds the max of 500KB.
 debg [navigation] state = "compiler success" after 374.3 sec
 debg [opensearchDashboardsOverview] state = "compiler success" after 381.7 sec
 debg [dataExplorer] state = "compiler success" after 397.2 sec
 debg [visAugmenter] state = "compiler success" after 428 sec
 debg [visualize] state = "compiler success" after 438.7 sec
 debg [management] state = "compiler success" after 439.4 sec
 debg [savedObjects] state = "compiler success" after 443.9 sec
 debg [savedObjectsManagement] state = "compiler success" after 446.9 sec
 debg [visDefaultEditor] state = "compiler success" after 455.7 sec
 debg [visualizations] state = "compiler success" after 460 sec
 debg [visTypeTimeline] state = "compiler success" after 490.4 sec
 debg [visBuilder] state = "compiler success" after 496.5 sec
 debg [home] state = "compiler success" after 515.5 sec
 debg [opensearchDashboardsReact] state = "compiler success" after 523 sec
 debg [visTypeTimeseries] state = "compiler success" after 541.8 sec
 debg [charts] state = "compiler success" after 546.9 sec
 debg [dashboard] state = "compiler success" after 560.3 sec
 debg [dataSourceManagement] state = "compiler success" after 576.1 sec
 debg [visTypeMetric] state = "compiler success" after 581.7 sec
 debg [core] state = "compiler success" after 592.4 sec
 debg [workspace] state = "compiler success" after 595 sec
 debg [securityDashboards] state = "compiler success" after 598.7 sec
 debg [discover] state = "compiler success" after 599.3 sec
 debg [indexPatternManagement] state = "compiler success" after 599.7 sec
 debg [usageCollection] state = "compiler success" after 610.3 sec
 debg [inputControlVis] state = "compiler success" after 612.1 sec
 debg [opensearchUiShared] state = "compiler success" after 631.4 sec
 debg [visTypeVega] state = "compiler success" after 631.8 sec
 debg [data] state = "compiler success" after 631.9 sec
 succ 54 bundles compiled successfully after 631.9 sec
```

Reply:
Looks like this number has gradually crept up over time. The biggest contributor in this list is `expressions`.